aip/general/0149.md

```diff
@@ -50,6 +50,15 @@ field behavior as defined in [AIP-203][]. For example, a field labeled with
 `google.api.field_behavior = REQUIRED` if the field must be set. If you only
 want to document the server perceived behavior of a field, read [AIP-203][].
 
+### Backwards compatibility
+
+Adding or removing the `optional` label to an existing field is considered a
+backwards incompatible change in some languages i.e. Go, due to a change in
```

Comment:
Nit: i.e. => e.g. (as there may be others too over time)

Reply:
Done!